stackstorm-morpheus/README.md

```diff
@@ -0,0 +1,70 @@
+# Morpheus Integration Pack
+This pack allows you to integrate with Morpheus
+
+## Configuration
+Copy the example configuration in [morpheus.yaml.example](./morpheus.yaml.example) to
+`/opt/stackstorm/configs/morpheus.yaml` and edit as required.
+
+It must contain:
+
+```
+ipaddress - Your Morpheus Appliance IP address
+username - Morpheus User name
+password - Morpheus Password
+dbuser  - Mongo database username
+dbpass - Mongo database password
+```
+
+You can also use dynamic values from the datastore. See the
+[docs](https://docs.stackstorm.com/reference/pack_configs.html) for more info.
+
+Example configuration:
+
+```yaml
+---
+  ipaddress: "10.10.10.10"
+  username: "Administrator"
+  password: "password"
+  dbuser: "appUser"
+  dbpass: "passwordForAppUser"
+```
+You can also run `st2 pack config morpheus` and answer the prompts
+
+**Note** : When modifying the configuration in `/opt/stackstorm/configs/` please
+           remember to tell StackStorm to load these new values by running
+           `st2ctl reload --register-configs`
+
+
+## Actions
+
+Actions are defined in two groups:
+
+### Individual actions: GET, POST, PUT with under bar will precede each individual action
```

Comment:
hank you for the questions and suggestions. I cleaned up the Action heading. Just a quick note on the architecture. When I read logs from Morpheus, I'm doing it on an interval timer. If the first time I pull 100 log entries and send them directly to Splunk, and then 5 minutes later I do it again, chances are I will get duplicate entries. I don't want to send anything to Splunk that I already sent, so I store them in a mongodb and mark them as unprocessed. Then when the splunk sender runs, it sets the flag to those same records as processed. When the splunk sender runs again, it will only pull the 'unprocessed' entries to Splunk avoiding any duplicate entries. At the time it was the simplest answer.